### ISSUE TITLE
SRS and $env:PSModulePath

### ISSUE THREAD

**Issue by lucdekens** [vmware/script-runtime-service-for-vsphere#27]
The $env:PSModulePath variable inside SRS seems to have some irregularities.

The content of the variable is currently
```
/root/.local/share/powershell/Modules:/usr/local/share/powershell/Modules:/app/service/Modules::/app/PowerCLIModules
```

- There is the double separator ([System.IO.Path]::PathSeparator which is colon on Linux) before **/app/PowerCLIModules** Is there a specific reason for that?
- The first three paths seem to point to non-existing modules directories

**Comment by dmilov** [comment]
No specific reason. The first three path are the default that come with powershell installation. The last one is appended by the runspace with path separator in front. The reason for double path separator is the last directory is appended with path separator in front.

Do you experience issues with this $PSModulePath?

**Comment by lucdekens** [comment]
I added some modules (following the procedure you outlined #19) before building the SRS OVA, andthat worked.

Then I tried to do an Install-Module from within a script I launched, which apparently worked as well.
The module was installed in folder **/root/.local/share/powershell/Modules/**
```
Module 'Posh-SSH' was installed successfully to path '/root/.local/share/powershell/Modules/Posh-SSH/2.3.0'
```
But the module doesn't seem to be available from any script I use.
I even did a restart of the SRS VM.

And when I try to list the contents of **/root/.local/share/powershell/Modules**, it comes up empty.

I was trying to find out why this happened, and that is where I stumbled on that odd $env:PSModulePath content.
